backend/src/kitconcept/intranet/locales/de/LC_MESSAGES/kitconcept.intranet.po

```diff
@@ -39,23 +51,47 @@ msgstr "Blockkonfiguration"
 msgid "Can only be seen and edited by the owner."
 msgstr "Kann nur vom Eigentümer angesehen und bearbeitet werden."
 
-#: kitconcept/intranet/controlpanels/intranet.py:31
+#: kitconcept/intranet/controlpanels/intranet.py:33
 msgid "Custom CSS"
 msgstr "Benutzerdefiniertes CSS"
 
-#: kitconcept/intranet/controlpanels/intranet.py:32
+#: kitconcept/intranet/controlpanels/intranet.py:34
 msgid "Custom CSS for the intranet."
 msgstr "Benutzerdefiniertes CSS für das Intranet."
 
+#: kitconcept/intranet/controlpanels/intranet.py:50
+msgid "Default Feedback Email"
+msgstr "Standard-Feedback-E-Mail"
+
+#: kitconcept/intranet/controlpanels/intranet.py:44
+msgid "Disabled"
+msgstr "Behindert"
```

Comment:
```suggestion
msgstr "Deaktiviert"
```

Though should this type of setting maybe be a checkbox instead of a dropdown?

Reply:
No Timo specially wanted that. It is related to another pr I just added the translation.